### PR TITLE
Diagnostics Author Info

### PR DESCRIPTION
**Pull request by rando2**
### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/metadata.yaml [hunk 1]
```diff
@@ -150,6 +151,12 @@ authors:
             city: Philadelphia
             state: PA
             country: USA
+      diagnosticss:
```
Comment (agitter) [inline content/metadata.yaml:154]:
```suggestion
      diagnostics:
```

File: content/metadata.yaml [hunk 2]
```diff
@@ -862,6 +896,12 @@ authors:
       lastapproved: !!str 2022-03-15
     funders:
       - Center for Effective Altruism, Long Term Future Fund
+    manuscripts:
+      diagnosticss:
```
Comment (agitter) [inline content/metadata.yaml:900]:
```suggestion
      diagnostics:
```

**Review by agitter — CHANGES_REQUESTED** [review]
I noticed two authors were missing in the docx.  This should fix it.

This might also be a good time to ask all authors to review and accept the code of conduct. From the build log:
```
Anna Ada Dattoli has not approved the code of conduct
Immunology Institute of the Icahn School of Medicine has not approved the code of conduct
Fengling Hu has not approved the code of conduct
Jinhui Wang has not approved the code of conduct
Missing contributions for Amruta Naik
Daniel S. Himmelstein has not approved the code of conduct
Jeremy P. Kamil has not approved the code of conduct
Jesse G. Meyer has not approved the code of conduct
Ariel I. Mundo has not approved the code of conduct
```

**Comment by rando2** [comment]
Thank you @agitter! I don't know how I did that and then also missed it while checking things through.

@aimundo @aadattoli @jessegmeyerlab Would you be willing to look at and approve the \[Code of Conduct\](https://github.com/greenelab/covid19-review/blob/master/CODE_OF_CONDUCT.md)?

**Comment by jessegmeyerlab** [comment]
> Thank you @agitter! I don't know how I did that and then also missed it while checking things through.
> 
> Authors edited in this (@aimundo @aadattoli @jessegmeyerlab) would you be willing to look at and approve the [Code of Conduct?](https://github.com/greenelab/covid19-review/blob/master/CODE_OF_CONDUCT.md)

I approve the code of conduct

**Comment by aimundo** [comment]
> Thank you @agitter! I don't know how I did that and then also missed it while checking things through.
> 
> Authors edited in this (@aimundo @aadattoli @jessegmeyerlab) would you be willing to look at and approve the [Code of Conduct?](https://github.com/greenelab/covid19-review/blob/master/CODE_OF_CONDUCT.md)

I approve the code of conduct

**Comment by rando2** [comment]
Thank you both! I don't think Ada sees her GitHub notifications but I'll check with her by email.

**Pull request merged**